### PR TITLE
Hydrate contributor data

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: resolwe/flow/serializers.py [hunk 1]
```diff
@@ -120,6 +152,7 @@ class CollectionSerializer(ResolweBaseSerializer):
     descriptor_schema = DescriptorSchemaSerializer(required=False)
 
     slug = serializers.CharField(read_only=False, required=False)
+    contributor = ContributorSerializer()
```
Comment (dblenkus) [inline resolwe/flow/serializers.py]: Add this to all serializers. Maybe to `ResolweBaseSerializer`?

**Comment by dblenkus** [comment]
Add change log entry and one minor comment. Otherwise LGTM.

**Comment by dblenkus** [comment]
And add tests for all possibilities in `to_internal_value` function

**Comment by kostko** [comment]
Done.

**Comment by kostko** [comment]
Just noticed that we can't really test the correctness of `to_internal_value` as `ResolweCreateModelMixin` always overrides the contributor value.

**Comment by kostko** [comment]
I've fixed tests so that we test the serializer's `to_internal_value` directly.

**Comment by mstajdohar** [comment]
LGTM. Please merge.

**Comment by hadalin** [comment]
@kostko, @mstajdohar Just noticed that `genialis-base/genesis/app_state/models/AppState` model also has a contributor filed, but does not extend resowle base model. Is this ok or should we leave it as it is.

**Comment by dblenkus** [comment]
Please, don't discuss private repos in public ones.

We decided a while ago that all models should be subclasses of `BaseModel`, so this should be handled automatically (serializers also should be subclasses of `ResolweBaseSerializer`). Only think that should be added is `.prefetch_related('contributor')` in viewset. But this is only for performance matter, would work anyway.

**Pull request merged**